code/drasil-code/Language/Drasil/Chunk/Code.hs

```diff
@@ -20,126 +20,161 @@ import qualified Data.Map as Map
 import Text.PrettyPrint.HughesPJ (render)
 
 -- not using lenses for now
+-- | Information that is needed to translate an idea into code. 
```

Comment:
This class is the other way around: "code as an idea". So it says that a `CodeIdea` must include both some code and its name.